tests/unittest_nodes.py

```diff
@@ -1583,6 +1583,7 @@ def func():
     """
     )
     assert node.doc == "Docstring"
+    assert node.doc_node
```

Comment:
It would be interesting to test more than that here. In particular information about the lineno, line_end, column, column end is important.

Reply:
Added tests; end line/column are optional and depend on the python version

Reply:
Could we also test (for example) the type of `node.doc_node`? And perhaps the value of the node?
You should also be able to add tests for `end_lineno`, either by creating a special test for `3.8`+ or with something like we do ourselves in `pylint` see:
https://github.com/PyCQA/pylint/blob/6caab13816e8098063b6e077ab0044b94aa87557/tests/testutils/test_output_line.py#L55-L62

Reply:
done